src/wallet/rpcwallet.cpp

```diff
@@ -2540,15 +2554,113 @@ static UniValue loadwallet(const JSONRPCRequest& request)
     return obj;
 }
 
-static UniValue createwallet(const JSONRPCRequest& request)
+static std::map<std::string,WalletFlags> walletflagmap{
```

Comment:
Can we define these in `wallet.h`, so there's only one place to change when adding wallet flags?